custom/icds_reports/templates/icds_reports/icds_app/mobile/mobile-filters.directive.html

```diff
@@ -4,8 +4,10 @@
         <div class="filter-header">
             <div class="d-flex justify-content-between align-items-center mb-3">
                 <div class="d-flex align-items-center">
-                    <img src="{% static "icds_reports/mobile/images/back-icon.png" %}" class="mr-2" ng-click="closeFilters()">
-                    <p class="font-montserrat font-weight-600 font-16 text-white mb-0">Filters</p>
+                    <nav-menu-opener ng-show="$ctrl.selectAwc"></nav-menu-opener>
```

Comment:
this shows the navigation menu over the filters when required to select an AWC?

Reply:
yes